zlib/README.md

```diff
@@ -0,0 +1 @@
+zlib/README
```

Comment:
Looks like you have added a symlink here? I think this isn't supported by the component manager now. Please add the basic description of the component here, instead. Please check other components in this repo for some examples.